src/stakingOptions.ts

```diff
@@ -348,6 +391,40 @@ export class StakingOptions {
     });
   }
 
+  /**
+   * Create a transaction for issue that includes setup instructions
+   */
+  public async createIssueTransaction(
+    amount: BN,
+    strike: BN,
+    name: string,
+    authority: PublicKey,
+    baseMint: PublicKey,
+    recipient: PublicKey,
+  ): Promise<Transaction> {
+    const optionMint = this.soMint(strike, name, baseMint);
+    const userSoAccount = getAssociatedTokenAddressSync(optionMint, recipient, true);
+    const transaction = new Transaction();
+
+    if (!(await this.connection.getAccountInfo(userSoAccount))) {
```

Comment:
Similar question as above, since we handle this on the UI side currently, do we want to move that here? I would expect to also see mention of a PR with the app changes?

https://github.com/Dual-Finance/dual-labs-dip/blob/2644636394b9324ae5b4d929be3acd194d3add44/app/src/utils/studio_contract.ts#L116